roles/matrix-maubot/tasks/main.yml

```diff
@@ -0,0 +1,126 @@
+---
```

Comment:
If one does `--tags=setup-all` now, this won't get configured, would it?

Likewise, when doing `--tags=start`, this won't be started, because it doesn't inject itself into `matrix_systemd_services_list` on an `always` tag.

Other roles split this out into an `init.yml` file, which executes on the `always` tag, like this: https://github.com/spantaleev/matrix-docker-ansible-deploy/blob/7c26d1b90e340c92b7ec5f5bb78da5af6cc26fe1/roles/matrix-ma1sd/tasks/main.yml#L1

And then `init.yml` takes care of the common setup (systemd service injection, nginx proxy configuration injection) like this: https://github.com/spantaleev/matrix-docker-ansible-deploy/blob/7c26d1b90e340c92b7ec5f5bb78da5af6cc26fe1/roles/matrix-bridge-mautrix-telegram/tasks/init.yml#L1

Would you split things into `main.yml`, `init.yml` and `setup.yml`?